playground/backend/containers/java/build.gradle

```diff
@@ -69,9 +69,9 @@ docker {
                   project.docker_image_default_repo_root)
   files "./build/"
   tags containerImageTags()
-  buildArgs(['BASE_IMAGE': project.rootProject.hasProperty(["base-image"]) ?
-                           project.rootProject["base-image"] :
-                           "apache/beam_java8_sdk" ])
+  buildArgs(['BEAM_VERSION': project.rootProject.hasProperty(["sdk-tag"]) ?
+                           project.rootProject["sdk-tag"] :
+                           "2.44.0" ])
```

Comment:
Can we avoid hard-coding the version?

Reply:
I didn't find good way to do this. Our approach for java runner is different from other runners that we can only use release beam version

Reply:
Could we at least pull it out as a constant at the top of the file?